include/lang.inc.php

```diff
@@ -53,7 +53,7 @@
     // Needs to be included here because we need access to constant LANG_CHARSET definied in languages (not available for compat.inc.php)
 
     if (function_exists('mb_language')) {
-        @mb_language($serendipity['lang']);
+        @mb_language('uni');
```

Comment:
I had to look up why you changed that. Could you add a comment?

Also, did I get this right? The old code was wrong because only a limited set of languages is allowed here, so you set it to uni - for unicode- if possible?

Reply:
It was crashing here, because I'm (still) i Spain and `@mb_language('es');` crashed. I think it's not the best solution because there are also non-uniode installations. Maybe a switch would be great.

Reply:
Would be interesting to see what this fell back to before! Then we could switch it to that, or to the correct values on non-utf8 enabled blogs. But shouldn't block this PR I think.